daemon/names.go

```diff
@@ -72,7 +72,7 @@ func (daemon *Daemon) reserveName(id, name string) (string, error) {
 				logrus.Errorf("got unexpected error while looking up reserved name: %v", err)
 				return "", err
 			}
-			return "", fmt.Errorf("Conflict. The container name %q is already in use by container %s. You have to remove (or rename) that container to be able to reuse that name.", name, id)
+			return "", fmt.Errorf("Conflict. The container name '%q' is already in use by container '%s'. You have to remove (or rename) that container to be able to reuse that name.", name, id)
```

Comment:
I'm not sure if this is the answer you are looking for but its added by stdib fmt https://github.com/golang/go/blob/964639cc338db650ccadeafb7424bc8ebb2c0f6c/src/fmt/format.go#L407-L422